extensions/ql-vscode/src/remote-queries/view/RepositoriesSearch.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { ChangeEvent } from 'react';
+import { TextInput } from '@primer/react';
+import { SearchIcon } from '@primer/octicons-react';
+
+interface RepositoriesSearchProps {
+  filterValue: string;
+  setFilterValue: (value: string) => void;
+}
+
+const RepositoriesSearch = ({ filterValue, setFilterValue }: RepositoriesSearchProps) => {
+  return <>
+    <TextInput
```

Comment:
In a large screen this looks a bit silly. Shall we set max-width? e.g. `max-width: 30em`.

<img width="1300" alt="image" src="https://user-images.githubusercontent.com/311693/160600934-e583bbbd-da06-4eac-baba-1f4dae2b38a9.png">

Reply:
Should we have a max-width for the entire results view and just let the search box follow that?

The wireframes have the search box exactly as wide as the NWO results.

Reply:
Good call!